ui/component/or-dashboard-builder/src/or-dashboard-boardsettings.ts

```diff
@@ -0,0 +1,275 @@
+import {Dashboard, DashboardAccess, DashboardScreenPreset, DashboardScalingPreset } from "@openremote/model";
+import {css, html, LitElement, TemplateResult } from "lit";
+import { customElement, property, state } from "lit/decorators.js";
+import { InputType, OrInputChangedEvent } from "@openremote/or-mwc-components/or-mwc-input";
+import {style} from './style';
+import { i18next } from "@openremote/or-translate";
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+import {
+    dashboardAccessToString,
+    scalingPresetToString,
+    sortScreenPresets
+} from ".";
+
+//language=css
+const boardSettingsStyling = css`
+    .label {
+        margin-bottom: 4px;
+    }
+`
+
+@customElement("or-dashboard-boardsettings")
+export class OrDashboardBoardsettings extends LitElement {
+
+    @property()
+    protected readonly dashboard?: Dashboard;
```

Comment:
Looks like this is always expected to be set so it shouldn't be `nullable`, maybe `dashboard!` would be more appropriate and then no need to use that syntax everywhere else.